src/renderer/components/Button.js

```diff
@@ -280,6 +280,7 @@ export const Base: ThemedComponent<*> = styled.button.attrs(p => ({
   cursor: ${p => (p.disabled ? "not-allowed" : "pointer")};
   height: ${p => (p.small ? 34 : 40)}px;
   pointer-events: ${p => (p.disabled ? "none" : "")};
+  width: ${p => (p.fullWidth ? "100%" : "auto")};
```

Comment:
Out of interest what is this doing? @achabrolin-ledger

Reply:
Basically it allow (via the `fullWidth` prop) to specify if the element should take the full with of his parent element (`100%`) or not. In the later case, the width would be calculated automatically by the browser due to the `auto` value

cf. https://developer.mozilla.org/en-US/docs/Web/CSS/width#values